From bcaa2f23e4ad775e2390a4a47cc0467ebb9cf788 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 15 Sep 2023 06:53:05 +0200
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index d7b8d24..fc85c19 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,7 +31,6 @@ requires-python = ">=3.9,<3.13"
 dependencies = [
      "pymatgen>=2024.5.1",
      "numpy<3.0.0",
-     "typing",
 ]
 
 [tool.setuptools.package-data]
